karspexet/ticket/payment.py

```diff
@@ -104,3 +96,30 @@ def _send_mail_to_customer(self):
             [to_address],
             fail_silently=False,
         )
+
+class FakePaymentProcess(PaymentProcess):
+
```

Comment:
Do we have the blank line at top of classes? Can't remember, but looks odd for some reason.